checkbox-ng/checkbox_ng/launcher/stages.py

```diff
@@ -453,9 +453,13 @@ def _override_exporting(self, export_fn):
         self._export_fn = export_fn
 
     def _prepare_stock_report(self, report):
+        try:
+            ConfigurationType = self.sa.configuration_type()
+        except AttributeError:
+            ConfigurationType = Configuration
```

Comment:
Add some comments about what this does and why this needs to catch this AttributeError exception.